apps/jobs/utils.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import status
+from rest_framework.response import Response
+from challenges.models import (
+    ChallengePhase,
+    Challenge,
+)
+from participants.models import (ParticipantTeam,)
+
+
+def get_challenge_object(challenge_pk):
+    try:
+        Challenge.objects.get(pk=challenge_pk)
+    except Challenge.DoesNotExist:
+        response_data = {'error': 'Challenge does not exist'}
```

Comment:
String should be something like `{model_name} {pk} does not exist`. Clearing this out so that no confusion arises later on!